tasks/section5.yml

```diff
@@ -251,15 +369,19 @@
       - rule_5.2.5
 
 - name: "SCORED | 5.2.5 | PATCH | Ensure SSH MaxAuthTries is set to 4 or less"
-  command: /bin/true
+  lineinfile:
+      state: present
+      dest: /etc/ssh/sshd_config
+      regexp: '^(#)?MaxAuthTries \d'
+      line: 'MaxAuthTries 4'
   tags:
       - level1
       - level2
       - patch
       - rule_5.2.5
 
 - name: "SCORED | 5.2.6 | AUDIT | Ensure SSH IgnoreRhosts is enabled"
-  command: /bin/true
+  command: grep "^IgnoreRhosts yes" /etc/ssh/sshd_config
   register: result
```

Comment:
Audit tasks should register verbose variable name that ends with `_audit`. As you are adding AUDIT checks please change `register: result` to `register: `*`descriptive_check_name_audit`*